### ISSUE TITLE
Comparison between Open Source and Paid Versions

### ISSUE THREAD

**Issue by cbailiss** [TabularEditor/TabularEditor#892]
I am trying to find a comparison table (features by version) that shows the differences between the open source and paid versions.  I might have missed it (looked in the Git README, in the docs and at https://tabulareditor.com/

I am guessing there are feature differences?  Or does the open source version have the full feature set but differences in support?

I think having a comparison explanation/table would be a good idea on the tabulareditor website as currently there is only a link to the open source version ("Looking for the open source version?") but nothing that explains why I should spend money on the paid version.

Sorry in advance if such a comparison exists and I have just missed it!

**Comment by otykier** [comment]
Good idea. We don’t currently have a table that compares the paid editions with the open source version, but you can find an overview of new features in the paid version here: https://tabulareditor.com/enterprise-edition/

To learn more about the differences between the 3 paid editions, see this article: https://docs.tabulareditor.com/projects/te3/en/latest/editions.html

**Comment by otykier** [comment]
**Update:** We actually do have a list of new features of Tabular Editor 3 compared to the open source version. Please see this: https://docs.tabulareditor.com/projects/te3/en/latest/getting-started.html#what-s-new-in-tabular-editor-3

**Comment by cbailiss** [comment]
Thanks for the info.  
From my perspective as a newcomer to Tabular Editor, the "list of features" page doesn't make clear to me that everything listed there is not in the open source version (is that the case?).  
The pages notes that:  "The following section assumes that you are somewhat familiar with the open-source Tabular Editor"...
... which I am not.

What I was looking for was something simple/clear like:
![image](https://user-images.githubusercontent.com/11745789/127973358-a0d66d96-22aa-42b7-8879-40044745fc0b.png)

So in the case of Tabular Editor, a table with five columns (the last four columns being ticks/crosses or similar):

- Features
- Open Source
- Desktop
- Business
- Enterprise

**Comment by otykier** [comment]
We have added a detailed comparison of TE2 and TE3 [here](https://docs.tabulareditor.com/index.html#choosing-between-te2-and-te3). Features are mostly identical across TE3 editions (Desktop/Business/Enterprise), with the exception of those features with an asterisk (*). These limitations are detailed [here](https://docs.tabulareditor.com/te3/editions.html).

Hope this helps!
